File: backend/models/quiz.js
const mongoose = require("mongoose");

const quizSchema = mongoose.Schema({
    questions: [
        {
            question: {
                type: String,
            },
            choices: [
                {
                    type: String,
                },
            ],
        },
    ],
    category: {
        type: String,
    },
});

const Quiz = mongoose.model("Quiz", quizSchema);

module.exports = Quiz;

File: alintana/app/src/main/java/com/mobdeve/s17/dizon/palmares/alintana/model/Quiz.kt
package com.mobdeve.s17.dizon.palmares.alintana.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Quiz : Serializable {
    @SerializedName("_id")
    var _id = ""

    @SerializedName("category")
    var category = ""

    @SerializedName("questions")
    var questions : ArrayList<Question>? = null

    constructor(_id: String, category: String, questions: ArrayList<Question>?) {
        this._id = _id
        this.category = category
        this.questions = questions
    }
}